app/controllers/MovementsController.scala

```diff
@@ -111,31 +104,32 @@ class MovementsController @Inject()(
                     InternalServerError
                   }
                 }
+            case Left(error) =>
+              Logger.warn(s"Failed to create ArrivalMovement with the following error: $error")
```

Comment:
I think it's worth properly showing the error. `Logger.error(msg, error)` or I am not sure if you can pass through an error object to the `Logger.warn` method, but we would need to, because in here we are throwing away the stack trace right now.